wp-modules/editor/model.php

```diff
@@ -116,16 +118,17 @@ function save_metadata_to_pattern_file( $override, $post_id, $meta_key, $meta_va
 				'post_name' => $meta_value,
 			]
 		);
+	}
 
-		if ( $pattern_name !== $meta_value ) {
-			delete_pattern( $pattern_name );
-		}
+	if ( $was_name_changed ) {
+		delete_pattern( $pattern_name );
```

Comment:
No functional change, just reduced the indent leve.

Reply:
update_pattern_slugs() from [PR 175](https://github.com/studiopress/pattern-manager/pull/175/files#diff-9cb8e65820f243200d64a962ca70432f543bfb448512c0744bed12e2b0bb6a12R167) can go here if you approve of this PR